### PR TITLE
[FLOC-4358] Release Flocker 1.11.0

### PR DESCRIPTION
**Pull request by wallrj**
Fixes: https://clusterhq.atlassian.net/browse/FLOC-4358

See: http://doc-dev.clusterhq.com/gettinginvolved/infrastructure/release-process.html#pre-tag-review-process

### CODE REVIEW

**Comment by zubron** [comment]
Thanks @wallrj. I made one commit just to fix the spelling of `Homebrew` in the release notes. I think we should also close #2716 as it hasn't been merged yet. I'll monitor the state of the builds for the rest of the day. Once they pass, continue with the rest of the release process.

File: docs/releasenotes/index.rst [hunk 1]
```diff
@@ -9,22 +9,26 @@ You can learn more about where we might be going with future releases by:
...
+  This allows larger clusters containing more datasets and supporting more frequent configuration changes.
+* Flocker can now be :ref:`installed on Redhat Enterprise Linux 7 <rhel-7-install-standalone-flocker>`.
 * The container agent is now optional and can be safely disabled if you don't expect to be using Flocker's deprecated container API or ``flocker-deploy``.
   The :ref:`Flocker plugin for Docker<plugin>` allows you to use Flocker from Docker without using Flocker's container API.
+* The dataset agent now has backend support for :ref:`Open vStorage <open-vstorage-backend>`.
+* A race condition where multiple volumes for a given dataset could be created and used has been fixed.
+  This could lead to the appearance of data loss, as different volumes get used.
+  Now, even if multiple volumes are created, only a single volume will be used.
+  This was particularly likely to occur on AWS.
+* The Flocker client tools can once again be installed on OSX Yosemite.
```
Comment (zubron) [inline docs/releasenotes/index.rst]: I had hit accepted but realised that `OSX` should be `OS X` as that's how it is elsewhere in our docs. I'll fix it.

**Comment by zubron** [comment]
The `jenkins-_run_sphinx_link_check` job is failing on this branch and on `master`. I've filed [FLOC-4360](https://clusterhq.atlassian.net/browse/FLOC-4360) to address this. I think that this should be fixed before this release is finished.

**Comment by wallrj** [comment]
I created a new cloudformation installer template and ran our tests on it. They pass

```
(flocker-1.11.0)(release/flocker-1.11.0 ✕? =)[~/.../HybridLogic/flocker-release-1.11.0]$ ACCEPTANCE_YAML=/home/richard/projects/HybridLogic/acceptance.yml CLOUDFORMATION_TEMPLATE_URL=https://s3.amazonaws.com/installer.downloads.clusterhq.com/flocker-cluster.cloudformation.1.11.0.json trial admin.test.test_installer
admin.test.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ...                                    [OK]

-------------------------------------------------------------------------------
Ran 1 tests in 782.238s

PASSED (successes=1)
```

**Pull request merged**